pkg/kubelet/util/queue/work_queue.go

```diff
@@ -46,6 +46,11 @@ func NewBasicWorkQueue() WorkQueue {
 	return &basicWorkQueue{queue: queue, clock: util.RealClock{}}
 }
 
+func NewTestWorkQueue(clock util.Clock) WorkQueue {
```

Comment:
Suggestion: add the `clock` arg to `NewBasicWorkQueue()` instead. We should not have test methods in this file.